ci/.env

```diff
@@ -10,6 +10,11 @@ IBIS_TEST_MYSQL_PORT=3306
 IBIS_TEST_MYSQL_USER=ibis
 IBIS_TEST_MYSQL_PASSWORD=ibis
 IBIS_TEST_MYSQL_DATABASE=ibis_testing
+IBIS_TEST_MSSQL_HOST=mssql
+IBIS_TEST_MSSQL_PORT=1433
+IBIS_TEST_MSSQL_USER=ibis
```

Comment:
probably you should use `sa` for user here

Reply:
Done.